src/pages/index.js

```diff
@@ -1,12 +1,21 @@
 import React from "react";
 
+import Navigation from "../components/Navigation";
+import Paragraph from "../components/Paragraph";
 import Title from "../components/Title";
 
 import "./index.css";
 
 const Pages = () => (
   <div>
     <Title>{`Design system`}</Title>
```

Comment:
Should we mention Barnardo's here?

Reply:
We had the same exact same thought and then decided against adding it because we anticipate the Barnardo's logo, in a future header, will give it context. This is how GDS [introduce](https://www.gov.uk/service-manual) their service manual.